docs/content/tutorial/backups.md

```diff
@@ -123,6 +128,57 @@ kubectl apply -k kustomize/s3
 
 Watch your cluster: you will see that your backups and archives are now being stored in S3!
 
+### Using an AWS-integrated identity provider and role
+
+If you deploy PostgresClusters to AWS Elastic Kubernetes Service, you can take advantage of their 
+IAM role integration. When you attach a certain annotation to your PostgresCluster spec, AWS will 
+automatically mount an AWS token and other needed environment variables. These environment 
+variables will then be used by pgBackRest to assume the identity of a role that has permissions 
+to upload to an S3 repository.
+
+This method requires additional setup in AWS, following the procedures outlined 
+[here](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts-technical-overview.html), 
+but skipping the third step:
```

Comment:
🤔 I see three steps on this page: https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html Is that where we should link?

⛏️ I prefer that link text not be just "here". Perhaps on "additional setup" or so?

```markdown
This method requires [additional setup in AWS IAM](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
Follow the procedure there, but skip the third step…
```